### ISSUE TITLE
Suggestion: Change "flip preview" to "flip webcam"

### ISSUE THREAD

**Issue by JohanAR** [alievk/avatarify#323]
I think it would be more useful if flip was applied immediately after reading a frame from the webcam, rather than only flipping the preview_frame.

The reason is that I don't have my webcam straight ahead of me, so my face is looking to the left (when watching the unflipped preview). This makes it very awkward to get a good match with an avatar that is facing right, and just flipping the preview doesn't change anything except how I see it, I still have to turn my head away from the screen to align with the avatar. But if the input frame is flipped instead, it will make the alignment easier in this case.

Alternatively another hotkey could be added to flip the avatar image, which would also solve this problem.

**Comment by alievk** [comment]
This is a suggestion for the GUI app, right?

**Comment by JohanAR** [comment]
Yes.. I could make a PR for it, but I wanted to check if you think it would be a good idea first

**Comment by alievk** [comment]
We've not published the sources of the GUI app yet. But we'll take this suggestion into account!

**Comment by JohanAR** [comment]
Ahh, I thought you meant cam_fomm.py when you said GUI app, I didn't know there was another.. For me (Linux user :)) everything that isn't a pure terminal app counts as GUI, even if it's just a video stream without buttons

**Comment by alievk** [comment]
Haha, strictly talking cam_fomm.py also has GUI :)

**Comment by xXCrowe3ye** [comment]
virtual camera is missing

**Comment by JohanAR** [comment]
I think maybe it wasn't clear exactly what I meant so I created PR #333 for it. You can reject it if you prefer the current behaviour
